app/assets/javascripts/controllers/automation_manager/automation_manager_form_controller.js

```diff
@@ -4,21 +4,19 @@ ManageIQ.angular.app.controller('automationManagerFormController', ['$http', '$s
   vm.automationManagerModel = angular.copy(configurationManagerService.managerModel);
 
   vm.postValidationModel = {};
-
-  vm.formId = automationManagerFormId;
+  vm.model = 'automationManagerModel';
   vm.afterGet = false;
   vm.saveable = miqService.saveable;
-  vm.validateClicked = configurationManagerService.validateClicked;
   vm.modelCopy = angular.copy(vm.automationManagerModel);
-  vm.model = 'automationManagerModel';
   vm.checkAuthentication = true;
+  vm.validationUrl = '/automation_manager/authentication_validate/' + automationManagerFormId + '?button=validate';
+  vm.prefix = 'default';
```

Comment:
Why this? Looks like the only place using it is in `validationStatus`, but you already have `attrs.prefix` there. Am I missing something?

Reply:
Ah, never mind, it's used in `validateClicked` here.

But I'm still confused about that `vm.prefix` versus `attrs.prefix` in `validationStatus`.

Reply:
Reason why vm.prefix is used only in one place is that i simply forgot to replace constants from template(it is done btw.).

vm.prefix and attrs.prefix are the same, but attrs.prefix is no longer available in auth-credentials component, because it used to be a parameter of the auth_credentials_angular_bootstrap partial. That is also reason, why i added that loop to find postvalidationregistryModel() to validation status, when the validation button used to be its own component (now its just a miq-button and the controller is handling the validation function). And since the prefix is not declared anywhere else i created vm.prefix attribute to store the prefix inside the controller.